test/jest/__snapshots__/NodeKey.spec.ts.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`NodeKey it should derive a child seed from private key 1`] = `"crouch cause bicycle tomato choice city produce ocean canyon rare laugh sudden close angle cage item special jeans either heavy skill bracket lift butter"`;
+exports[`NodeKey it should derive a child seed from private key 1`] = `"chat toward myself element estate miracle merit pudding next net process idle mobile hawk transfer chimney border gate patient artefact square strike worth already"`;
```

Comment:
I'm curious why this snapshot update is necessary? Maybe I'm missing something, but there are no changes to `NodeKey.ts`

Reply:
We use the name of the swap client in the entropy used to generate the child seed, so it makes sense that this is changing because it's going from raiden to connext.

Reply:
Ah, right. It's because the enum changed. Thanks for clearing this up.